prime-dotnet-webapi/Services/OrganizationService.cs

```diff
@@ -381,5 +384,29 @@ public async Task RemoveUnsignedOrganizationAgreementsAsync(int organizationId)
             _context.RemoveRange(pendingAgreements);
             await _context.SaveChangesAsync();
         }
+
+        // update organization registration ID calling OrgBook API with organization name in PRIME
+        public async Task<int> UpdateOrganizationRegistrationId()
+        {
+            var targetOrganizations = await _context.Organizations.Where(o => o.RegistrationId == null)
+                .OrderBy(o => o.Id)
+                .ToListAsync();
+            int counter = 0;
+            if (targetOrganizations.Any())
+            {
+                foreach (var org in targetOrganizations)
+                {
+                    string registrationId = await _orgBookClient.GetOrgBookSearchRecordAsync(org.Name);
+                    if (registrationId != null)
+                    {
+                        org.RegistrationId = registrationId;
+                        counter++;
+                    }
```

Comment:
Unfortunately can't use `CreateOrganizationEventAsync` without `partyId`.  That said, although the logs are ephemeral, should log to INFO that an organization's registration ID was updated by this job.

Reply:
Log info is added.